### ISSUE TITLE
Turning off Android Keyboard Prediction on program load

### ISSUE THREAD

**Issue by luwizghieherald** [EyeSeeTea/UICMalawi#2]
Hi team,

I was going through the app, I would like to know if it is possible to turn off the android keyboard prediction feature when the program is opened. Looking at the UIC being so sensitive, the users must enter the required parameters without the prediction so that they are sure of what they are punching in.

Since most people are used to the error-prone and fast typing using the prediction feature, it might contribute to some unforeseen erroneous data to be captured.

Thanks.

**Comment by ifoche** [comment]
Hi @luwizghieherald ,

we will study the thing. It's highly possible that we don't find a generic solution, because keyboards might be different from one device to another. And sometimes there are models that use their own keyboard (like samsung ones) which are not fully compatible with standard Android ones. In any case, we will try to figure out if we can give you a solution that works for most of the cases so you can test

**Comment by ifoche** [comment]
We will try to provide you with a proposal for removing automatically keyboard suggestions from the app. However, keep in mind that keyboard suggestions are something that the user can manage from the device settings.

**Comment by luwizghieherald** [comment]
> We will try to provide you with a proposal for removing automatically keyboard suggestions from the app. However, keep in mind that keyboard suggestions are something that the user can manage from the device settings.

That is true, or can we have a popup message (splash screen) to notify or remind users that auto-fill is not recommended when using the app?
